### PR TITLE
Improve matmul (small changes)

### PR DESCRIPTION
**Pull request by okuta**
This PR is a part of #1547. Please merge this PR first.

### CODE REVIEW

**Comment by okuta** [comment]
jenkins, test this please.

File: tests/cupy_tests/math_tests/test_matmul.py [hunk 1]
```diff
@@ -20,6 +21,7 @@
             ((3, 0), (0,)),
             ((2,), (2,)),
             ((0,), (0,)),
+            #matmul test
```
Comment (toslunar) [inline tests/cupy_tests/math_tests/test_matmul.py]:
```suggestion
            # matmul test
```

**Comment by okuta** [comment]
I fixed.

**Comment by kmaehashi** [comment]
Jenkins, test this please.

**Comment by toslunar** [comment]
The Jenkins failure is unrelated to the PR.

**Pull request merged**